package-lock.json

```diff
@@ -24,7 +24,7 @@
         "ts-jest": "^27.0.5",
         "typedoc": "^0.22.18",
         "typedoc-plugin-merge-modules": "^3.1.0",
-        "typescript": "^4.7.4"
```

Comment:
You need to change this on `package.json` and not `package-lock.json`. Then you can delete `package-lock.json` and regenerate it locally. Then you can change `package.json` back (before committing) and `package-lock.json` should be fixed.

Can explain later after meeting if it would help.